sympy/polys/numberfields/tests/test_subfield.py

```diff
@@ -277,6 +277,10 @@ def test_primitive_element():
     assert primitive_element([sqrt(2), 0], x, ex=True) == (x**2 - 2, [1, 0], [[MPQ(1,1), MPQ(0,1)], []])
     assert primitive_element([0, sqrt(2)], x, ex=True) == (x**2 - 2, [1, 1], [[], [MPQ(1,1), MPQ(0,1)]])
 
+    # Issue 27798
+    a, b = CRootOf(49*x**3 - 49*x**2 + 14*x - 1, 2), CRootOf(49*x**3 - 49*x**2 + 14*x - 1, 0)
+    assert primitive_element([a, b], polys=True)[0].primitive()[0] == 1
+    assert primitive_element([a, b], polys=True, ex=True)[0].primitive()[0] == 1
 
 def test_to_number_field():
```

Comment:
There should be two blank lines between the test functions here and it is better to have a URL for the issue like:
```
https://github.com/sympy/sympy/issues/27798
```
That makes it clear that it is a GitHub issue number and also is easy to click and go straight to the issue.